src/compositemark/errorbar.ts

```diff
@@ -340,10 +349,29 @@ export function errorBarParams<
     encodingWithoutContinuousAxis,
     ticksOrient: orient === 'vertical' ? 'horizontal' : 'vertical',
     markDef,
-    outerSpec
+    outerSpec,
+    tooltipEncoding
   };
 }
 
+export function errorBarTooltipEncoding(
+  continuousAxisChannelDef: PositionFieldDef<string>,
+  encodingWithoutContinuousAxis: Encoding<string>
+): Encoding<string> {
+  const tooltopSummaryMap: CompositeMarkTooltipSummary[] = [
+    {fieldPrefix: 'upper', titlePrefix: 'Upper error'},
```

Comment:
Title "Upper/lower error" is confusing -readers won't get whether the error bar encodes Stdev, Stderr, CI. (It's basically confusing.) 

For pre-aggregated case, better not provide special title and use original field name. 

For aggregating case, we should provide information about what error bar presents.  For example, we could  say `mean(x)+stdev(x) = ...`, `mean(x) - stdev(x)` = ...Alternatively, we could just directly set `titlePrefix: titlecase(aggregate)` in the tooltip and show the error instead of upper/lower (so we get `Stdev of X`) and reduce the complex +/- thing. 

cc: @domoritz  Any thoughts

Reply:
It might be worth looking at how other systems display tooltips for error bars too.  

I suspect that some might even says  "10 ± 5".  So another alternative is 
`Mean of x ± Stdev of X:   10 ± 5`.   

Right now I think this might be the best, but would require some custom signal that we couldn't do without a signal / expression support yet.